acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/LotusNode.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.dsl.lotus;
+
+import java.io.IOException;
+
+import com.github.arteam.simplejsonrpc.client.JsonRpcClient;
+import com.google.common.net.MediaType;
+import org.apache.commons.codec.Charsets;
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+/**
+ * Lotus is expected to be running outside of acceptance test. It can be executed using following
+ * docker command <code>
+ *     docker run --rm --name lotuslocalnet -e TEXLOTUSDEVNET_SPEED=1500 \ -e
+ * TEXLOTUSDEVNET_BIGSECTORS=false -p 1234:7777 \ -v /tmp/import:/tmp/import textile/lotus-devnet
+ * </code>
+ */
+public class LotusNode {
+  private static final String FC_URL_FORMAT = "http://%s:%d/rpc/v0";
+  private final String fcUrl;
+  private final JsonRpcClient jsonRpcClient;
+
+  public LotusNode(final String host, final int port) {
+    fcUrl = String.format(FC_URL_FORMAT, host, port);
+    jsonRpcClient = initClient();
+  }
+
+  public LotusNode(final int port) {
+    this("127.0.0.1", port);
+  }
+
+  public void loadDefaultAddresses() {
+    AddressesUtil.getDefaultFilecoinAddressMap()
+        .forEach(
+            (address, key) -> {
+              if (!hasAddress(address)) {
+                importPrivateKey(key);
+              }
+            });
+  }
+
+  public JsonRpcClient getJsonRpcClient() {
+    return jsonRpcClient;
+  }
+
+  private void importPrivateKey(final FilecoinKey key) {
+    jsonRpcClient
```

Comment:
nit: Not sure if its possible, but would be AWESOME if we could have a generic "request builder" then we send literally the same request to both Eth2Signer and Lotus (in fact, in some ways, we don't even know its lotus, its just an endpoint).

Reply:
I achieved it in the base comparisonapi class ... by feeding in jsonRpcClient. This particular method only imports it in FileCoin and will not currently work for Eth2Signer.

Reply:
Refactored out FilecoinJsonRequests.